packages/forma-36-react-components/src/components/CheckboxField/README.mdx

```diff
@@ -0,0 +1,64 @@
+# CheckboxField
+
+CheckboxField is containing a checkbox with its label. Use this component to give users the possibility to select multiple elements, zero, one or more items. It can be used in forms or terms and conditions. For a single choice, please consider using RadioButton component. 
```

Comment:
I'm still in favour of this order:

1. "when to use"
2. what it consists of
3. (optional) when to not use it, but rather use the radio button

Reply:
@domarku oki! 😁 WDYT if we do this:
"Use this component to give users the possibility to select multiple elements, zero, one or more items. It can be used in forms or terms and conditions. CheckboxField is containing a checkbox with its label. For a single choice, please consider using RadioButton component."